distribution/src/assemble/bin.xml

```diff
@@ -345,6 +345,16 @@
       <fileMode>0755</fileMode>
       <outputDirectory>conf</outputDirectory>
     </file>
+    <file>
+      <source>src/resources/drill-auto.sh</source>
```

Comment:
"-auto"? Is this for the self-driving car feature of Drill? :-)

Is this meant to be for startup-time environment checks? For other add-on processing? Perhaps we can pick a name that makes that a bit clearer.

Reply:
AutoConfig was the intent, but looked a bit verbose. How about `drill-autocheck.sh` ? :)

Reply:
Modifying the base commit (#1081) to reflect the name change from `[distrib/drill]-auto.sh` to `[distrib/drill]-setup.sh`